tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/PitSummaryView.java

```diff
@@ -36,6 +38,9 @@ public PitSummaryView(SWTWorkbenchBot bot) {
     public void getViewIfNotSet() {
         if (summaryView == null) {
             summaryView = (PitView) Views.getViewById(PitView.VIEW_ID);
+            if (summaryView != null) {
```

Comment:
Of course that's a typo: you meant == ;)

Reply:
Thanks. My head hurt already, from not knowing why it is failing.